test/index.html

```diff
@@ -60,6 +60,12 @@ <h1>Hello, world!</h1>
 
             <p><a class="btn btn-primary btn-lg btn-success" href="#" role="button">Continue</a></p>
 
+            <article id="article">
+                <p>This paragraph is visible.</p>
+                <p id="p-with-hidden">This paragraph has a <span id="span" aria-hidden="true">hidden</span> span.</p>
+                <p id="hidden-p" aria-hidden="true">This paragraph is hidden.</p>
+            </article>
```

Comment:
This isn't currently working

![image](https://user-images.githubusercontent.com/1266297/57953731-91fa6d00-78b6-11e9-97ea-a068de425d2b.png)

I turn on the visualizer but they aren't getting hidden (also, can this text say `aria-hidden` instead of `hidden` so it's clear what we mean?

Perhaps it's just that this draft doesn't do that bit yet. Sorry if I'm jumping ahead here.

Reply:
That was an error on my part. Should be working as expected after the next `push`.